recipes/lastpass-cli/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "lastpass-cli" %}
+{% set version = "1.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lastpass/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e6f9a5df9fab46cb6192f40762933faba002479d9387eecbd5ffb28efb9d88b
+  patches:
+    # Need patch from https://github.com/lastpass/lastpass-cli/issues/532
+    - 532-fix-common.patch
+
+build:
+  number: 0
+  script: "make && make install"
+  # Skip windows build, but upstream supports windows
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - asciidoc  # [not win]
+    - libxml2   # [not win]
+    - cmake
+    - make
+    - pkgconfig
+  host:
+    - libcurl
+    - libxml2
+    - openssl
+  run:
```

Comment:
You should be able to completely omit this section as they all should come in via run_exports